.circleci/config.yml

```diff
@@ -25,7 +25,7 @@ jobs:
       - run: git ls-files '*.html' | xargs pipenv run djhtml --check
       - run: pipenv run curlylint --parse-only wagtail
       - run: pipenv run doc8 docs
-      - run: DATABASE_NAME=wagtail.db pipenv run python -u runtests.py
```

Comment:
Not sure if this was intentional (so that we use a file instead of an in-memory database), but we're not using `--keepdb` so I don't think there's much value in this.

Functionality-wise, there shouldn't be any difference in between the two. From https://www.sqlite.org/inmemorydb.html:

> Even though a disk file is allocated for each temporary database, in practice the temporary database usually resides in the in-memory pager cache and hence there is very little difference between a pure in-memory database created by ":memory:" and a temporary database created by an empty filename. The sole difference is that a ":memory:" database must remain in memory at all times whereas parts of a temporary database might be flushed to disk if the database becomes large or if SQLite comes under memory pressure.

Reply:
Thanks! Interestingly, none of our CI configs (or our local `Makefile`/pre-commit configs) seem to use `tox`. It is mentioned in our contributing guide, though.